db/db_impl.h

```diff
@@ -223,13 +229,15 @@ class DBImpl : public DB {
   volatile size_t current_block_size_;    // last dynamic block size computed
   volatile uint64_t block_size_changed_;  // NowMicros() when block size computed
   volatile uint64_t last_low_mem_;        // NowMicros() when low memory last seen
+  volatile bool hotbackup_pending_;       // true if hotbackup cycle initiated, blocks close
```

Comment:
I'll shift stuff as long as it does not cause grief in the constructor (which it should not).